src/users/register.js

```diff
@@ -14,17 +14,47 @@ export function register(user /** async */) {
     url         : Urls.userRegister(),
     isAsync     : isAsync,
     asyncHandler: responder && wrapAsync(responder),
-    data        : user
+    data        : enrichWithLocaleInfo(user)
   })
 
   return isAsync ? result : parseResponse(result)
 }
 
+function enrichWithLocaleInfo(user) {
+  if (user.blUserLocale === null || user.blUserLocale === '') {
+    return user
+  }
+
+  user.blUserLocale = getClientLanguage()
```

Comment:
also need to describe the property in the types definitions file https://github.com/Backendless/JS-SDK/blob/master/backendless.d.ts#L142

Reply:
https://github.com/Backendless/JS-SDK/pull/138/commits/8236d55a663159a9ec52e0e73d762ab864c351ea